fineract-core/src/main/java/org/apache/fineract/batch/service/BatchApiServiceImpl.java

```diff
@@ -94,6 +94,68 @@ public class BatchApiServiceImpl implements BatchApiService {
     @PersistenceContext
     private final EntityManager entityManager;
 
+    /**
+     * Run each request root step in a separated transaction
+     *
+     * @param requestList
+     * @param uriInfo
+     * @return
```

Comment:
Please do not reorder the content of the classes because it is nearly impossible to see what was changed... Please revert to the original order and change only what need to be changed, but not the order of the methods!

Reply:
Set it to the logical order so I was able to follow the process. the name of the methods did not change.